src/amdgpu.cpp

```diff
@@ -49,6 +49,7 @@ bool amdgpu_verify_metrics(const std::string& path)
 }
 
 #define MAX(x, y) (((x) > (y)) ? (x) : (y))
+#define IS_VALID_METRIC(FIELD) (FIELD != 0xffff)
```

Comment:
Is there another method we could determine if it's valid or not?
My concern is that it can be valid and still be 0

Reply:
The last mail I received from the freedesktop mail list was the following:
```
[AMD Official Use Only - General]

For some members, "0" is a valid value. 
Thus "0xffff" is used instead to tell the output is invalid/unsupported.

BR
Evan
```
So I think that this is the best method for determining if it is valid or not.

Reply:
Yeah sounds great then :+1:

Reply:
Here is a link to that email: https://lists.freedesktop.org/archives/amd-gfx/2023-February/089461.html

In case you are interested.